keylime/tpm/tpm2_objects.py

```diff
@@ -440,3 +441,26 @@ def ek_low_tpm2b_public_from_pubkey(pubkey: pubkey_type) -> bytes:
         EK_LOW_AUTH_POLICY,
         EK_LOW_NON_ASYM_ALG_PARMS,
     )
+
+
+def parse_tpms_clock_info(clock_info: bytes) -> Dict[str, int]:
+    (clock, resetCount, restartCount, safe) = struct.unpack(">QIIB", clock_info)
+    return {"clock": clock, "resetCount": resetCount, "restartCount": restartCount, "safe": safe}
+
+
+def get_tpms_attest_clock_info(tpms_attest: bytes) -> Dict[str, Any]:
+    (
+        magic,
+        _,
+    ) = struct.unpack_from(">IH", tpms_attest, 0)
```

Comment:
nit: but I wonder why black is doing this, when `magic, _ =  struct.unpack_from(">IH", tpms_attest, 0)` seems shorter that other lines

Reply:
don't try to overrule `black` ...

Reply:
yeah, it does not matter

Reply:
I don't know why black did this...

Reply:
there is a chance that it is a bug somewhere, here it is working tho:

```
# pip list | grep black
black           23.3.0

# cat kk.py
import struct


def get_tpms_attest_clock_info(tpms_attest):
    magic, _ = struct.unpack_from(">IH", tpms_attest, 0)

# black --diff kk.py 
All done! ✨ 🍰 ✨1 file would be left unchanged.
```